evaluation/datasets/common.py

```diff
@@ -73,11 +73,12 @@ def _check_dataset_on_s3(self):
     @property
     def s3_client(self):
         if self._s3_client is None:
+            region = getattr(self.task, "aws_region", None) or self._config["aws_region"]
```

Comment:
I'm a bit confused why we would need to add the aws task region here to make it worked for decentralized, since as of right now the task owner does not have access to change the task region anyway